src/controllers/graphql/connectors/connectors.js

```diff
@@ -3,7 +3,7 @@ const scheduler = require('node-schedule');
 
 const db = require('../../../models/sequelize');
 
-db.sequelize.sync({force: false}).then(() => {
+db.sequelize.sync({force: true}).then(() => {
```

Comment:
This as well. #21 
These are no problem because they're the same so there won't be merge conflicts, but you just did twice the work haha